g2p_registry_rest_api/services/individual_helper.py

```diff
@@ -0,0 +1,108 @@
+import json
+
+from odoo import models
+
+
+class IndividualApiHelper(models.AbstractModel):
+    _name = "registrant_individual.rest.service.helper"
+
+    def _get(self, _id):
+        partner = self.env["res.partner"].browse(_id)
+        if partner and partner.is_registrant and not partner.is_group:
+            return partner
+        return None
+
+    def process_individual(self, individual):
+        indv_rec = {
+            "name": self.process_name(
+                individual.family_name, individual.given_name, individual.addl_name
+            ),
```

Comment:
This should be in the specific implementation using calculated fields.

Reply:
This is just reused from [here](https://github.com/OpenG2P/openg2p-registry/blob/df459d140e1232fbc6249cfc2652fd527a6b4798/g2p_registry_individual/models/individual.py#L27).

Reply:
Yes, but you should not call this method yourself, it is automatically called when you are going to save the data.